test/e2e/e2e-agent-reboot/pom.xml

```diff
@@ -84,6 +84,7 @@
                             <run>
                                 <env>
                                     <INSTRUMENTED_SERVICE>${project.build.finalName}.jar</INSTRUMENTED_SERVICE>
+                                    <INSTRUMENTED_SERVICE_OPTS>-DSW_AGENT_COOL_DOWN_THRESHOLD=1</INSTRUMENTED_SERVICE_OPTS>
```

Comment:
How this opt works? If you use `-D`, then it should be `skywalking.agent. cool_down_threshold`, right?

Your own document says so too
> `agent.cool_down_threshold `|How long should the agent wait (in minute) before re-registering to the OAP server after receiving reset command.|`10`|